[MIG] l10n_ar_tax_python: Migration to 19.0

migration-bot-adhoc opened the pull request.

migration-bot-adhoc committed:
[MIG] l10n_ar_tax_python: Migration to 19.0

roboadhoc commented:
[![Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina/pull/1232.png)](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina/pull/1232)

jjscarafia committed:
[REF] l10n_ar_tax: cleanup of invisible fields

jjscarafia committed:
[FIX] l10n_ar_tax_python: finish migration

nocopy no longer needed https://github.com/odoo/odoo/commit/e73f638d5f63b40ad0db1c6237a3bef85524ef63

jjscarafia commented:
@roboadhoc r+ rebase-ff

roboadhoc commented:
Merge method set to rebase and fast-forward.
